apps/framework-cli/src/analytics/posthog.rs

```diff
@@ -1,75 +1,62 @@
 use chrono::{DateTime, Utc};
+use posthog514client_rs::{Event514, EventType, PostHogClient as BasePostHogClient, PostHogError};
 use serde_json::json;
+use std::collections::HashMap;
 
 use crate::cli::settings::Settings;
 
 // Build-time environment variable for PostHog API key
 const POSTHOG_API_KEY: Option<&str> = option_env!("POSTHOG_API_KEY");
 const POSTHOG_HOST: &str = "https://us.i.posthog.com";
 
-#[derive(Debug, thiserror::Error)]
-pub enum PostHogError {
-    #[error("Failed to send event to PostHog: {0}")]
-    RequestError(#[from] reqwest::Error),
-    #[error("PostHog API key not configured")]
-    ApiKeyMissing,
-}
-
-#[derive(Debug, Clone)]
-pub struct PostHogEvent {
-    pub event: String,
-    pub distinct_id: String,
-    pub timestamp: DateTime<Utc>,
-    pub properties: serde_json::Value,
-}
-
 #[derive(Clone)]
 pub struct PostHogClient {
```

Comment:
Why did you decide to keep this here? vs moving it to the lib?

Reply:
I had a small wrapper that I was using in moose - I've removed that wrapper in favor of direct use.